src/libraries/Common/tests/TestUtilities/System/TestSecretHelper.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    public static partial class TestSecretHelper
```

Comment:
Is this just a magic word to the tool?

Reply:
It is just the ordinary name of the class. It should give a hint to the developer it annotates safe credentials.